carsus/io/kurucz/gfall.py

```diff
@@ -13,6 +13,7 @@
 
 
 GFALL_URL = 'https://media.githubusercontent.com/media/tardis-sn/carsus-db/master/gfall/gfall_latest.dat'
```

Comment:
Also, why keep this if it won't be used? What do others think of this?

Reply:
It can be removed

Reply:
removed